[2024-08-25T08:42:18.324+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-25T08:42:18.363+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_catchup_and_backfill_v2.task1 backfill__2024-08-19T00:00:00+00:00 [queued]>
[2024-08-25T08:42:18.378+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_catchup_and_backfill_v2.task1 backfill__2024-08-19T00:00:00+00:00 [queued]>
[2024-08-25T08:42:18.381+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-08-25T08:42:18.413+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): task1> on 2024-08-19 00:00:00+00:00
[2024-08-25T08:42:18.425+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1801) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-25T08:42:18.427+0000] {standard_task_runner.py:72} INFO - Started process 1810 to run task
[2024-08-25T08:42:18.427+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_catchup_and_backfill_v2', 'task1', 'backfill__2024-08-19T00:00:00+00:00', '--job-id', '74', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_and_backfill.py', '--cfg-path', '/tmp/tmp63hj_kfc']
[2024-08-25T08:42:18.434+0000] {standard_task_runner.py:105} INFO - Job 74: Subtask task1
[2024-08-25T08:42:18.536+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_catchup_and_backfill_v2.task1 backfill__2024-08-19T00:00:00+00:00 [running]> on host ae952724060e
[2024-08-25T08:42:18.695+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='shreyas' AIRFLOW_CTX_DAG_ID='dag_with_catchup_and_backfill_v2' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-08-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2024-08-19T00:00:00+00:00'
[2024-08-25T08:42:18.698+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-25T08:42:18.720+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-25T08:42:18.722+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo This is a simple bash command!']
[2024-08-25T08:42:18.739+0000] {subprocess.py:86} INFO - Output:
[2024-08-25T08:42:18.741+0000] {subprocess.py:93} INFO - This is a simple bash command!
[2024-08-25T08:42:18.743+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-25T08:42:18.796+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-25T08:42:18.797+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_and_backfill_v2, task_id=task1, run_id=backfill__2024-08-19T00:00:00+00:00, execution_date=20240819T000000, start_date=20240825T084218, end_date=20240825T084218
[2024-08-25T08:42:18.878+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-25T08:42:18.909+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-25T08:42:18.914+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
